<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MajorWindow</class>
 <widget class="QMainWindow" name="MajorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MajorWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">#widget{
border-image: url(:/my_image/3.gif);
}</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>250</y>
         <width>131</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/f6.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>270</y>
         <width>111</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/g8.ico);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>260</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/q10.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>390</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>温度</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>390</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>风力</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>390</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>方正粗黑宋简体</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>天气</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>180</y>
         <width>41</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>智
能
家
居
环
境
信
息
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>1130</x>
         <y>10</y>
         <width>81</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/Batman.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>30</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/g1.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>20</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/g12.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>580</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/i3.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>570</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/Vinyl Blue.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>580</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/Computer_001.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>570</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/m9.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>690</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>视频监控</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>680</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>多媒体</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>680</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>设备管理</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>1020</x>
         <y>680</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>游戏娱乐</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>20</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 18pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>显示日期</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_4">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>140</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/login/loging_button_bj.png);
color: rgb(0, 0, 0);
font: 16pt &quot;Arial&quot;;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>武汉</string>
       </property>
      </widget>
      <widget class="QLabel" name="TempLabel">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>440</y>
         <width>161</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 127);
font: 15pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>温度显示</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>输入城市名</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="WindLabel">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>440</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 127);
font: 15pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>风力显示</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="TypeLabel">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>440</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 127);
font: 15pt &quot;方正粗黑宋简体&quot;;
</string>
       </property>
       <property name="text">
        <string>天气类型</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <zorder>pushButton</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>pushButton_3</zorder>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>pushButton_4</zorder>
      <zorder>pushButton_5</zorder>
      <zorder>pushButton_6</zorder>
      <zorder>pushButton_7</zorder>
      <zorder>pushButton_8</zorder>
      <zorder>pushButton_9</zorder>
      <zorder>pushButton_10</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>label_7</zorder>
      <zorder>label_8</zorder>
      <zorder>label_4</zorder>
      <zorder>label_12</zorder>
      <zorder>lcdNumber_4</zorder>
      <zorder>lineEdit</zorder>
      <zorder>TempLabel</zorder>
      <zorder>label_9</zorder>
      <zorder>WindLabel</zorder>
      <zorder>TypeLabel</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
